src/lib/project.ts

```diff
@@ -56,32 +43,22 @@ export interface ProjectInfo {
 
 export interface Tree {
```

Comment:
AFAIK we don't return `kind` in heartwood, so we can remove it

See httpd json build fn https://github.com/radicle-dev/heartwood/blob/9d244b221a167c1c2bacbd00ecefc1a44ecab552/radicle-httpd/src/api/json.rs#L68

Reply:
eeacc38feec9059cfc5653449c3f7f39e2e312c9